apps/dashr-multipage-report/app.R

```diff
@@ -0,0 +1,1916 @@
+                  showgrid = FALSE,
+                  showline = TRUE,
+                  tickfont = list(size = 10), 
+                  ticks = "outside", 
+                  ticksuffix = " ", 
+                  title = "", 
+                  type = "linear",
+                  zeroline = TRUE,
+                  zerolinecolor = "rgb(190, 191, 192)"
+                )
+              )
+            )
+          )
+        ), className = "six columns", style = list('height' = "250px"))
+      ), className="thirdPage row", style = list('margin-top' = "30px"))
+    ), className = "subpage")
+  ), className = "page"),
+  
+  # Page 6
+  
+  htmlDiv(list(
+    
+    htmlDiv(list(
+      
+      htmlDiv(list(
+        htmlP(LipP(1, F), style = list('float' = "left"), 
+              className = "six columns"),
+        htmlP(LipP(1, F), style = list('float' = "left", 'margin-left' = "5px"), 
+              className = "six columns")
+      ), className = "thirdPage row", style = list('position' = 'relative', 'margin-top' = '40px')),
+      
+      htmlDiv(list(
+        htmlStrong("Vehicula elementum congue penatibus massa, eu sed", className = "eleven columns",
+                   style = list('color' = color_1)),
+        htmlDiv(
+          htmlImg(src = "http://i.imgur.com/DBkxRT2.png", 
+                  style = list('margin-top' = "20px", 'height' = "250px"))
+        )
+      ), className = "eleven columns"),
+      
+      htmlDiv(list(
+        htmlStrong("At velit pharetra ac fusce sit dictum pellentesque", className = "eleven columns", 
+                   style = list('color' = color_1)),
+        htmlDiv(list(
+          dccGraph(
+            figure = list(
+              data = list(
+                list(
+                  x = wtiPrices$`WTI Prices, x`,
+                  y = wtiPrices$`WTI Prices, y`,
+                  line = list(
+                    color = color_1, 
+                    dash = "solid"
+                  ), 
+                  mode = "lines", 
+                  name = "WTI Prices", 
+                  type = "scatter", 
+                  uid = "1380d4"
+                ),
+                list(
+                  x = wtiPrices$`Sep-15 forecast, x`,
+                  y = wtiPrices$`Sep-15 forecast, y`,
+                  line = list(color = "rgb(228, 31, 35)"), 
+                  mode = "lines", 
+                  name = "Sep-15 forecast", 
+                  type = "scatter", 
+                  uid = "766493"
+                ),
+                list(
+                  x = wtiPrices$`Forward, x`,
+                  y = wtiPrices$`Forward, y`,
+                  line = list(
+                    color = color_2, 
+                    dash = "solid"
+                  ), 
+                  mode = "lines", 
+                  name = "Forward", 
+                  type = "scatter", 
+                  uid = "4e8248"
+                )
+              ),
+              list(
```

Comment:
As it is, this leads to 
```
Failed component prop type: Invalid component prop `figure` key `2` supplied to Graph. Bad object: { "2": { "x": [ ...
```

This last trace needs to be placed within the `data` of the `figure` list to work:

```
          dccGraph(
            figure = list(
              data = list(
                list(
                  x = wtiPrices$`WTI Prices, x`,
                  y = wtiPrices$`WTI Prices, y`,
                  line = list(
                    color = color_1, 
                    dash = "solid"
                  ), 
                  mode = "lines", 
                  name = "WTI Prices", 
                  type = "scatter", 
                  uid = "1380d4"
                ),
                list(
                  x = wtiPrices$`Sep-15 forecast, x`,
                  y = wtiPrices$`Sep-15 forecast, y`,
                  line = list(color = "rgb(228, 31, 35)"), 
                  mode = "lines", 
                  name = "Sep-15 forecast", 
                  type = "scatter", 
                  uid = "766493"
                ),
                list(
                  x = wtiPrices$`Forward, x`,
                  y = wtiPrices$`Forward, y`,
                  line = list(
                    color = color_2, 
                    dash = "solid"
                  ), 
                  mode = "lines", 
                  name = "Forward", 
                  type = "scatter", 
                  uid = "4e8248"
                ),
                list(
                  x = wtiPrices$`May-15 forecast, x`,
                  y = wtiPrices$`May-15 forecast, y`,
                  line = list(
                    color = color_3,
                    dash = "solid"
                  ), 
                  mode = "lines", 
                  name = "Forward", 
                  type = "scatter"
                )
              ),
              layout = list(
                height = 250, 
                hovermode = "closest", 
                legend = list(
                  ...

```